src/appnap.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_APPNAPINHIBITOR_H
```

Comment:
Please don't add these kind of one-use utilities in the top-level src directory. As I've suggested before, a an `src/support` or such directory would make sense.